package-lock.json

```diff
@@ -3,6 +3,10980 @@
   "version": "1.1.0",
   "lockfileVersion": 1,
```

Comment:
With node 16 & npm 8 we should actually get lockfileVersion 2
Please regenerate package-lock with correct npm version.